src/hooks/useENSProfileForm.ts

```diff
@@ -98,7 +97,7 @@ export default function useENSProfileForm({
 
   // Set initial records in redux depending on user input (defaultFields)
   useEffect(() => {
-    if (defaultFields && isEmpty(defaultRecords)) {
+    if (defaultFields && !isEmpty(defaultRecords)) {
```

Comment:
don't we want to fill the defaults when the records are empty?

Reply:
just changed it!

Reply:
reverted back per video sent via slack

Reply:
are we still reverting this?